R/prep_alignment_table.R

```diff
@@ -27,7 +27,8 @@ prep_alignment_table <- function(results_portfolio,
     match.arg(asset_class)
     check_data_prep_alignment_table(scenario_source = scenario_source)
 
-    # infer start_year
+    if (nrow(results_portfolio %>% filter(asset_class == .env$asset_class)) > 0) {
```

Comment:
NIT/style: I prefer not having pipes inside expectations/ function calls etc. I usually define the object outside of the expectation and then test it.

Reply:
or if you don't want to persist the object, just `chain(the(functions))` rather than `functions %>% the %>% chain`